FrameworklessServer/Controllers/Controller.cs

```diff
@@ -37,8 +37,11 @@ public Response HandleRequest(Request request)
                 case "/users" when request.Method == "DELETE":
                     return DeleteUser(ReadBody(request.Body).Name);
             }
+
             if (System.Text.RegularExpressions.Regex.IsMatch(request.Path, $"/users/{Regex}"))
-                return GetNameFromUrl(request.Path);
+            {
+                return request.Method == "PUT" ? UpdateUser(request.Path.Split("/")[2], ReadBody(request.Body).Name) : GetNameFromUrl(request.Path);
```

Comment:
Feels a bit odd for `PUT` alone to not be a part of the switch block. Do you think its fixable? All thoughts are welcome.